src/main/java/com/google/sps/daos/DatabaseGameDao.java

```diff
@@ -83,4 +83,18 @@ public Game getGame(String id) {
       }
       return newGame;
     }
+
+
+@Override
+    public String getQuestionId(String question, String gameId, String currentQuestionId) {
```

Comment:
`nextQuestionId` and `previousQuestionId` could be fields on `Question`. So when you have a reference to the current question, just call `currentQuestion.nextQuestionId()` and update the `GameInstance` with that value.

Reply:
I think that's more less what I'm doing here, I would still have to get the question from the db and `getNextQuestion()` right? 
In here I'm doing the same but I'm just doing `document.get("nextQuestion").toString();`
Correct me if i'm wrong maybe I didn't understood